linera-views/tests/reentrant_tests.rs

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use linera_views::{
+    memory::create_test_context,
+    reentrant_collection_view::ReentrantCollectionView,
+    register_view::RegisterView,
+    views::{CryptoHashRootView, RootView, View},
+};
+use rand::{Rng, SeedableRng};
+use std::collections::{BTreeMap, BTreeSet};
+
+#[derive(CryptoHashRootView)]
+pub struct StateView<C> {
+    pub v: ReentrantCollectionView<C, u8, RegisterView<C, u32>>,
```

Comment:
If you don't want to name the field `view` or `inner`, then the expected syntax would be
```
pub struct StateView<C>(ReentrantCollectionView<C, u8, RegisterView<C, u32>>);
```

Does it not work with the derive macro?

Reply:
It does not work, the macro panicked. Created the issue #744